README.md

```diff
@@ -149,8 +149,6 @@ Our goal is to provide an always up-to-date documentation from within the app.
 
 Documentation is available at http://YOUR-HOST/docs
 
-> v1 docs are not maintained anymore and are just available as static swagger file
```

Comment:
Yeah for now I decided it's perhaps time to completely get rid of api/v1. It's something we can talk about though, it just seemed like it would make the process of updating a lot easier.